src/components/common/DateTime/index.tsx

```diff
@@ -1,11 +1,16 @@
 import { ReactElement } from 'react'
 import { Tooltip } from '@mui/material'
 import { formatDateTime, formatTimeInWords } from '@/utils/date'
+import { useRouter } from 'next/router'
+import { AppRoutes } from '@/config/routes'
 
 const DAYS_THRESHOLD = 60
 
 const DateTime = ({ value }: { value: number }): ReactElement => {
-  const displayExactDate = Math.floor((Date.now() - value) / 1000 / 60 / 60 / 24) > DAYS_THRESHOLD
+  const router = useRouter()
+  const isHistory = router.pathname.includes(AppRoutes.transactions.history)
```

Comment:
This will match filters as well.

Reply:
Yes, filtered results are also historical. This matches `safe-react`.

Reply:
I guess what I meant is unrelated to this line. I meant that maybe we don't need to show the full date when it's in the date label.

Reply:
Also why is it `includes` and not just a strict comparison?

Reply:
> I meant that maybe we don't need to show the full date when it's in the date label.

Do you mean something like this?

```tsx
<span>{filter ? formatDateTime(value) : displayExactDate ? formatTime(value) : formatTimeInWords(value)}</span>
```

>Also why is it `includes` and not just a strict comparison?

I'll adjust it.

Reply:
Filter and queue, yes. They both don't have date labels.